classification.yaml

```diff
@@ -0,0 +1,791 @@
+DMS-MWBT-REQ-0018:
+  status: needs-service
+  priority: 1a
+  script:
+  - Use https butler to run QG generation against an RSP user repo from outside RSP.
+  - Use https butler to transfer QG inputs to a new local repo.
+  - Run QG against local repo.
+  missing:
+  - Transfer support in https butler.
+  - User data repositories in RSP.
+  - https butler service that can access user data repositories in RSP.
+  reuse_script: DMS-MWBT-REQ-001
+DMS-MWBT-REQ-0020:
+  status: feature-complete
+  priority: 1a
+  script:
+  - Make empty repo.
+  - Run `butler register-skymap`.
+  jira: LVV-T1982
+DMS-MWBT-REQ-0021:
+  status: feature-complete
+  priority: 2
+  script:
+  - Make empty repo.
+  - Run `butler register-skymap`.
+  - Run `butler register-skymap` again.
+  reuse_script: DMS-MWBT-REQ-0020
+DMS-MWBT-REQ-0080:
+  status: feature-complete
+  priority: 1a
+  script:
+  - Run `butler query-datasets` against any major repo (e.g. `/repo/main`).
+DMS-MWBT-REQ-0081:
+  status: feature-complete
+  priority: 1a
+  script:
+  - Run `butler query-datasets` against any major repo (e.g. `/repo/main`), with multiple input collections.
+DMS-MWBT-REQ-0082:
+  status: feature-complete
+  priority: 1a
+  script:
+  - Run `butler query-datasets` against any major repo (e.g. `/repo/main`), with multiple input collections that contain the same unresolved DatasetRefs, with findFirst=False.
+DMS-MWBT-REQ-0087:
+  status: feature-complete
+  priority: 1a
+  script:
+  - Run `butler query-datasets` against any major repo (e.g. `/repo/main`), with multiple input collections that contain the same unresolved DatasetRefs, with findFirst=True.
+  notes:
+    Requirements description makes it clear that findFirst logic is considered adequate for satisfying this requirement, even if that's not obvious from the specification itself.
+DMS-MWBT-REQ-0088:
+  status: feature-complete
+  priority: 1a
+  script:
+  - Run `butler query-datasets` against any major repo (e.g. `/repo/main`), with a WHERE expression involving some dimension metadata fields.
+DMS-MWBT-REQ-0089:
+  status: feature-incomplete
+  priority: 1a
+  script:
+  - Run `butler query-datasets` against any major repo (e.g. `/repo/main`), with a WHERE expression involving some metric values.
+  missing:
+  - Tabular-data Datastore, used to store metrics, backed by opaque tables.
+  - Support for query expressions that reference opaque tables.
+DMS-MWBT-REQ-0090:
+  status: feature-incomplete
+  priority: 1a
+  script:
+  - Run `butler query-datasets` against any major repo (e.g. `/repo/main`), with an uploaded dataset ID constraint.
+  missing:
+  - Upload and query support for lists of dataset IDs.
+DMS-MWBT-REQ-0091:
+  status: unclear
+  priority: 1b
+  notes:
+    This is something we have long talked about adding, but haven't needed yet, and it's not the way we are currently planning to satisfy the linked use cases.
+    We should consider a change request to remove this requirement.
```

Comment:
I think I lean towards removing this requirement on "per dataset metadata" (with per-StorageClass or per-DatasetType schema), because it's not clear we need and requirements are for things we clearly need.

We may still add that functionality in the future as the best way to solve some problem.

Reply:
I admit that reading this I was wondering what we really wanted -- reading the entire FITS header into some butler database seems like a bad idea. For now we have dimension metadata solving the immediate problem of people wanting to query on important exposure information.

Reply:
How about we change this to P3 and revise it again if we get a use case.